### DISCUSSION TITLE
How to use use --editable to skip cleaning for the packages you specify in CMakeLists.txt file

### DISCUSSION THREAD

**Discussion by ilongshan** [microsoft/vcpkg#26139] [Q&A]
How to use --editable to skip cleaning for the packages you specify in  CMakeLists.txt file?

**Comment by dg0yt** [comment]
Assuming that you refer to manifest mode (which implicitly invokes vcpkg), the CMake variable is `VCPKG_INSTALL_OPTIONS`:
https://github.com/microsoft/vcpkg/blob/master/docs/users/buildsystems/cmake-integration.md#vcpkg_install_options

**Reply by ilongshan** [reply]
@dg0yt The option --editable is not supported in manifest mode (#16874). Thanks for your reply.

**Reply by dg0yt** [reply]
Thanks for the pointer!
